contrib/builder/deb/generate.sh

```diff
@@ -92,6 +104,40 @@ for version in "${versions[@]}"; do
 
 	echo >> "$version/Dockerfile"
 
+	# debian jessie & ubuntu trusty/vivid do not have a libseccomp.a for compiling static dockerinit
+	# ONLY install libseccomp.a from source, this can be removed once dockerinit is removed
+	case "$suite" in
+		jessie|trusty|vivid)
+			awk '$1 == "ENV" && $2 == "SECCOMP_VERSION" { print; exit }' ../../../Dockerfile >> "$version/Dockerfile"
+			cat <<-EOF >> "$version/Dockerfile"
```

Comment:
I don't see any actual environment variables from the enclosing environment used here; couldn't we switch this to use `<<-'EOF'` and drop a few `\` on all those `$`?